funnel/views/participant.py

```diff
@@ -125,10 +125,9 @@ def participant(profile, space):
     if space.date_upto:
         if midnight_to_utc(space.date_upto + timedelta(days=1), space.timezone, naive=True) < datetime.utcnow():
             return jsonify(message=u"This event has concluded", code=401)
-    participant = Participant.query.filter_by(puk=request.args.get('puk')).first()
-    if not participant:
-        return jsonify(message=u"Not found", code=404)
-    elif participant.key == request.args.get('key'):
+    form = ParticipantContactExchangeForm(parent=space)
```

Comment:
If you keep support for `GET`, you can't use a form. Since the form doesn't actually do anything more than a query, use `@requestargs` on this view instead, and just put the query in the view. Form is unnecessary.